include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -17,19 +17,26 @@
 #include <utility>
 #include <vector>
 
-#include <seqan3/alignment/configuration/all.hpp>
 #include <seqan3/alignment/matrix/detail/alignment_score_matrix_one_column.hpp>
 #include <seqan3/alignment/matrix/detail/alignment_score_matrix_one_column_banded.hpp>
 #include <seqan3/alignment/matrix/detail/alignment_trace_matrix_full.hpp>
 #include <seqan3/alignment/matrix/detail/alignment_trace_matrix_full_banded.hpp>
-#include <seqan3/alignment/pairwise/policy/all.hpp>
+#include <seqan3/alignment/pairwise/policy/affine_gap_policy.hpp>
```

Comment:
Hm, I really wonder if this is better than the all header since we can probably be sure that every policy will be needed in the alignment_configurator anyway...

Reply:
Alright